test/dovetail/programs_test.rb

```diff
@@ -7,83 +7,75 @@
   EPISODES = {
     'criminal' => 'prod_criminal/23166e58-e181-4c03-b52d-6f6746a1bced/Episode_35__Pen_and_Paper.mp3',
     'memory'   => 'prod_memory/c9dfaa10-7b21-4804-9d5e-7f55dfbefc24/thememorypalace.mp3',
-    'serial'   => 'prod_serial/b10f2326-b3b9-478e-a74d-22c460946316/serial-s02-e05.mp3',
+    # 'serial'   => 'prod_serial/b10f2326-b3b9-478e-a74d-22c460946316/serial-s02-e05.mp3',
```

Comment:
Serial isn't on cloudfront yet, and there aren't a whole ton of other programs that have identical episode guids between staging and prod.